### DISCUSSION TITLE
Automatically set post to expire if they have a tag

### DISCUSSION THREAD

**Discussion by ambivalentmaybe** [publishpress/publishpress-future#348] [Feature Requests]
I have some posts in one category I would like to automatically set to expire--they are weekly digests of events that have a shelf life of a month, at most. But other posts in that same category can be more long-lived. 

These weekly digest posts are, however, set apart by a tag. Unfortunately, I don't see a way in the present plugin to expire posts by tag. That would be a useful feature, though.

**Comment by andergmartins** [comment]
Related to #341
